content/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Adding_Additional_Authentication_Factors.adoc

```diff
@@ -0,0 +1,199 @@
+== Adding an additional authentication factor implementation guidance
+
+Most FIDO2 authenticators come packaged as two authentication factors: something you have (which is typically the authenticator), and either a something you know (like a device PIN), or something you are (like a biometric scanner). There are some legacy CTAP/U2F authenticators that are backwards compatible with WebAuthn, but are typically missing the something you know or something you are factor, often referred to as ‘user verification’ (UV).  The guidance given on this page will teach you how to implement an additional factor through your application for users missing UV on their authenticators.
+
+This scenario works best when:
+
+* You want to allow users to continue to register with legacy CTAP/U2F authenticators
```

Comment:
I think the goal here is that the relying party wants to offer the passwordless experience and still support legacy U2F devices.
Consider rewording to something like this:
* You want to give your users a passwordless experience and also support legacy CTAP/U2F authenticators
* You require MFA and want to support both the WebAuthn Passwordless experience and legacy CTAP/U2F devices